claudio-plugin/skills/jira-gap-audit/SKILL.md

```
@@ -0,0 +1,94 @@
+**Options:**
+- `--project KEY` — Jira project key (required)
+- `--post-release` — also check for Post-Release Activities epic
+
+**Examples:**
+```bash
+./scripts/audit_release.sh --project MYPROJ MyProduct "3.4 GA" --post-release
+./scripts/audit_release.sh --project MYPROJ MyProduct "3.3.2"
+```
+
+**Output:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)